next/services/graphql/queries/sections.gql

```diff
@@ -84,7 +84,7 @@ fragment AccordionSection on ComponentSectionsAccordion {
     ...FlatText
   }
   tableRows(pagination: { limit: -1 }) {
-    ...TableRow
+    ...TableRowWithId
```

Comment:
Other parts of code is also using `TableRow` and it will be unnecessary for then. This way every component gets exactly what it needs and nothing more.